content/tutorial-paychan.md

```diff
@@ -0,0 +1,489 @@
+# Payment Channels Tutorial
+
+Payment Channels are an advanced feature for sending "asynchronous" XRP payments that can be divided into very small increments and settled later. This tutorial walks through the entire process of using a payment channel, with examples using the [JSON-RPC API](reference-rippled.html) of a local `rippled` server.
+
+1. **The payer creates a payment channel to a particular recipient.** In doing so, the payer pre-funds the channel with XRP.
+
+    This is a [PaymentChannelCreate transaction][]. As part of this process, the payer sets certain specifics of the channel like an expiration time and a settlement delay, which affect the guarantees around the claims in the channel. The payer also sets the public key that will be used to verify claims against the channel.
+
+    **Tip:** The "settlement delay" does not delay the settlement, which can happen as fast as a ledger version closes (3-5 seconds). The "settlement delay" is a forced delay on closing the channel so that the payee has a chance to finish with settlement.
```

Comment:
But it's only defined in the Tip, which still seems a little backward to me. Shouldn't it be defined before it's clarified? Currently, it's only introduced in a kind of vague way.